lib/games_teams.rb

```diff
@@ -13,7 +13,7 @@ def initialize(data)
     @shots = data[:shots]
     @tackles = data[:tackles]
     @pim = data[:pim]
-    @powerPlayOpportunities = data[:powerplayopportunities]
+    @powerplayopportunities = data[:powerplayopportunities]
```

Comment:
uniformity capitalization across files seems to be our most common enemy holding us up